STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -269,7 +269,10 @@ In order to indicate which specific delegation applies to the requested
 certificate a new `delegation` attribute is added to the Order object on the
 NDC-IdO side (see {{sec-profile-ndc-to-ido}}).  The value of this attribute is
 the URL pointing to the delegation configuration object that is to be used for
-this certificate request.
+this certificate request.  If the `delegations` attribute in the Order object
+contains a URL that does not correspond to an existing configuration, the IdO
```

Comment:
to an existing configuration which is available to this client (or something like that: not all delegations can be used by all NDCs).

Reply:
thanks, fixed in 6caa3eb